.github/workflows/build-push-kubeai.yml

```diff
@@ -17,6 +17,8 @@ env:
 jobs:
   kubeai:
     runs-on: ubuntu-latest
+    # Timeout to address Github actions stuck scenarios 
+    timeout-minutes: 10
```

Comment:
Let's increase this to 60 minutes because the action can take up to 20 minutes in normal case.

Reply:
I added a commit that does this.